lib/node/fullnode.js

```diff
@@ -147,6 +144,14 @@ class FullNode extends Node {
       noAuth: this.config.bool('no-auth')
     });
 
+    // Index needs access to the node.
```

Comment:
Is this needed for some of the events?

Reply:
Yup, we need it to listen to the chain events. This specific comment can be improved though, will update it.